packages/ui/client/components/views/ViewEditor.vue

```diff
@@ -120,13 +244,22 @@ async function onSave(content: string) {
 </script>
 
 <template>
-  <CodeMirror
-    ref="editor"
-    v-model="code"
-    h-full
-    v-bind="{ lineNumbers: true }"
-    :mode="ext"
-    data-testid="code-mirror"
-    @save="onSave"
-  />
+  <div>
+    <IconButton
+      v-if="testIndex !== null || isDescribeBlock !== null || selectedTest !== null"
```

Comment:
Can we create a computed property that holds this to avoid duplicate.

Reply:
Sure